src/org/tensorflow/yolo/view/ClassifierActivity.java

```diff
@@ -136,11 +184,65 @@ private void renderAdditionalInformation(final Canvas canvas) {
             }
         }
 
-        lines.add("Frame: " + previewWidth + "x" + previewHeight);
-        lines.add("View: " + canvas.getWidth() + "x" + canvas.getHeight());
-        lines.add("Rotation: " + sensorOrientation);
-        lines.add("Inference time: " + lastProcessingTimeMs + "ms");
+// shimatani
+        // lines.add("Frame: " + previewWidth + "x" + previewHeight);
+        // lines.add("View: " + canvas.getWidth() + "x" + canvas.getHeight());
+        // lines.add("Rotation: " + sensorOrientation);
+        // lines.add("Inference time: " + lastProcessingTimeMs + "ms");
+
+        Log.d(LOGGING_TAG, String.format("検出名: %s", nowRecognizedClass));
+        lines.add("検出名: " + nowRecognizedClass);
+
+        String msgInf = makeTts(nowRecognizedClass);
```

Comment:
Where is the `msgInf` value used?

Reply:
Certainly, it was not used anywhere.

At first, I wanted to display a single `msgInf` with a line break, but I did not know how to do so, so I decided to display two lines, `msgInf1` and `msgInf2`.